sensu-plugins-elasticsearch.gemspec

```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |s|
   s.version                = SensuPluginsElasticsearch::Version::VER_STRING
 
   s.add_runtime_dependency 'rest-client',       '1.8.0'
-  s.add_runtime_dependency 'elasticsearch',     '~> 1.0.14'
+  s.add_runtime_dependency 'elasticsearch',     '>= 1.0.10'
```

Comment:
This will pull in the latest version. I suggest we cap it at a major version so we don't get burned later. Also, do we know for sure the 5.x gem will work fine against ES 2.x?